src/Agent.Plugins/PipelineArtifact/PipelineArtifactPlugin.cs

```diff
@@ -24,6 +25,8 @@ public async Task RunAsync(AgentTaskPluginExecutionContext context, Cancellation
             // TODO: Translate targetPath from container to host (Ting)
             string targetPath = context.GetInput(ArtifactEventProperties.TargetPath, required: true);
 
+            this.tracer = new CallbackAppTraceSource(str => context.Output(str), System.Diagnostics.SourceLevels.Information);
```

Comment:
does this cause any output on a call without retries?

Reply:
I don't think so.